packages/gatsby-source-wordpress/src/steps/create-schema-customization/helpers.js

```diff
@@ -188,3 +258,44 @@ export const filterTypeDefinition = (
 
   return typeDefinition
 }
+
+// we should be using graphql-js for this kind of thing, but unfortunately this project didn't use it from the beginning so it would be a huge lift to refactor to use it now. In the future we will be rewriting this plugin using a new Gatsby source plugin toolkit, and at that time we'll use graphql-js.
+// from introspection field types this will return a value like:
+// `String` or `[String]` or `[String!]!` or `[String]!` or `[[String]]` or `[[String]!]!` or `[[String]!]`, etc
```

Comment:
the comment says it all. this was needed to properly generate type strings for new wpgql fields